ui/src/react-admin/modules/content-page/types/content-block.types.ts

```diff
@@ -156,6 +156,7 @@ export enum ContentBlockType {
 	HetArchiefHeaderSearch = 'HETARCHIEF__HEADER_SEARCH',
 	OverviewNewspaperTitles = 'OVERVIEW_NEWSPAPER_TITLES',
 	ContentEncloseGrid = 'CONTENT_ENCLOSE_GRID',
+	Breadcrumbs = 'BREADCRUMBS',
```

Comment:
does this BREADCRUMBS block also have to be added to the hasura using a migration? in the lookup block types table?

Reply:
No it is already in the database